infra-as-code/bicep/main.bicep

```
@@ -44,16 +44,16 @@ param frontendCloudInitAsBase64 string
 @secure()
 param adminPassword string
 
-@description('The Azure Active Directory group/user object id (guid) that will be assigned as the admin users for all deployed virtual machines.')
+@description('The Microsoft Entra group/user object id (guid) that will be assigned as the admin users for all deployed virtual machines.')
```

I all previous instances the replacement name included ID at the end. Not sure if it was an oversight

```suggestion
@description('The Microsoft Entra ID group/user object id (guid) that will be assigned as the admin users for all deployed virtual machines.')
```

---

According to the product docs, it's "Microsoft Entra group" or "Microsoft Entra user"

So, if I have "ID" in another place... I need to fix it there.

---

You are good then. Just noticed the other instances are referencing the product in general, not a specific group or id